Blog build

ralph-rutter opened the pull request.

ralph-rutter committed:
Adding .gitignore file.

ralph-rutter committed:
Adding all starting files.

ralph-rutter committed:
Adding new page template for blog

ralph-rutter committed:
Added new button in navbar for blog page

ralph-rutter committed:
Adapting "next/previous" nav buttons for blog page

ralph-rutter committed:
Added content container to blog.php and adjusted z-indices

ralph-rutter committed:
Changed size and rounded corners of blog container

ralph-rutter committed:
Added title and subtitle to blog

ralph-rutter committed:
Added archive button and adjusted subtitle

ralph-rutter committed:
Making <a> white, only in nav, added links to blog content.

ralph-rutter committed:
Added html structure for hlinked items on blog page.

ralph-rutter committed:
Made msqli query and outputted to 'most recent'

ralph-rutter committed:
Reformatted appearance of links in blog most recent

ralph-rutter committed:
fixed some links

ralph-rutter committed:
Exchanged manually typed list of links for a foreach loop

ralph-rutter committed:
Changed 'desc' to 'synopsis' in mysqli code

ralph-rutter committed:
Moved databse connection code to connect.php and included it

ralph-rutter committed:
Functionalised and docblocked the database query script

ralph-rutter committed:
Functionalised foreachloop in most recent

ralph-rutter committed:
Changed database name and added DocBlock to links_most_recent

aimee-wyld commented on about.html:
```diff
@@ -0,0 +1,110 @@
+	<main>
+		<div class="container margin">
+			<div class="annex pre">
+				<img src="images/structure-1473346_1920.jpg alt="fractal"">
```
you've got an extra quote mark after fractal

ralph-rutter replied:
Amended in commit 634af97e20935afb990fc844a63e96c416d632ea

ralph-rutter replied:
Amended my incorrect amendment, in commit b6636e7a7136e6ded2f0b58baa296c29f9f9ec1c

ralph-rutter committed:
Removed extra quote mark in fractal img

ralph-rutter committed:
Putting in missing quote mark

aimee-wyld commented on css/top-bar.css:
```diff
@@ -52,4 +54,11 @@
+}
+
+#top-bar .content-container {
+    background: white;
```
might be worth specifying the property as background-color? & also might be worth using RGB or hex values (even though it's only white)?

aimee-wyld commented on css/top-bar.css:
```diff
@@ -64,4 +64,58 @@
+
+#top-bar .header {
+    position: relative;
+    background: #55286f;
```
again, maybe worth specifying background-color? But not sure about this!

aimee-wyld commented on css/top-bar.css:
```diff
@@ -64,4 +64,58 @@
+    -moz-border-radius: 0 0 18px 18px;
+    -webkit-border-radius: 0 0 18px 18px;
+    border-radius: 0 0 18px 18px;
+    font-size: 0px;
```
Don't need to put in px if it's 0

aimee-wyld commented on css/top-bar.css:
```diff
@@ -64,4 +64,58 @@
+    margin-top: -25px;
+    margin-left: -47px;
+    font-size: 38px;
+    background: #55286f;
```
Same background-color but again not sure what the standard is

aimee-wyld commented on css/top-bar.css:
```diff
@@ -64,4 +64,58 @@
+#top-bar .header h3 {
+    font-size: 20px;
+    margin: 0;
+    color: orange;
```
RGB/hex?

aimee-wyld commented on css/top-bar.css:
```diff
@@ -64,4 +64,58 @@
+    font-size: 20px;
+    margin: 0;
+    color: orange;
+    background: black;
```
RGB/hex and background-color?

ralph-rutter committed:
Changing 'background' to 'background-color' in multiple place

aimeeew reviewed (approved):
All looks great :)

ralph-rutter committed:
Created test file, wrote good test for links_most_recent()

also changed the indentation on the original function to make test work.

ralph-rutter committed:
Added bad and malform tests.

ralph-rutter committed:
Changed styling to overflow auto on blog.php

ralph-rutter committed:
Created 'include' folder and created functions.php

ralph-rutter committed:
Added functions for the blog archive page

ralph-rutter committed:
Moved and renamed a file storing potentially useful code

ralph-rutter committed:
added database_draft.php, a holding place for code

ralph-rutter committed:
added a font .DS_Store to .gitignore

ralph-rutter committed:
Added a new function call and switch to make page dynamic

ralph-rutter committed:
Added functions for indiv. article page

ralph-rutter committed:
Added a new case to switch and made subtitle dynamic

ralph-rutter committed:
Added links file and 'required' it on all pages

ralph-rutter committed:
Added a switch for the mysqli query and commented php

aimeeew reviewed (approved):
Looks very accurate, can't see any changes :)

ralph-rutter committed:
Added remainder of unit tests (all passed)

ralph-rutter committed:
Added cross-browser compatibility for a border-radius

ralph-rutter committed:
Replaced occurrences of '0px' with '0'

aimee-wyld commented on tests/functions.php:
```diff
@@ -0,0 +1,220 @@
+<?php
+
+require('../include/functions.php');
```
I don't think you need brackets for require because it's a statement rather than a function (googled it yesterday when I wasn't sure!!)

ralph-rutter replied:
Thanks, I've taken these out and committed vvv

ralph-rutter committed:
Removed some unneccesary brackets.

ralph-rutter commented:
With regards to your comments from yesterday and this morning:
- I've replaced any colours specified as a word with the corresponding hexadecimal value. 
- I've replaced all occurrences of 'background: _colour_' with 'background-color: ...'
- I've replaced all occurrences of '0px' with '0'
- I've kept the 'big-button' class, as, upon reflection, it will only ever be used for that purpose (hopefully!)

ralph-rutter committed:
Changing db conn. lib. from MySQLi to PDO

ralph-rutter committed:
Changed a colour word for a hex. value.

ralph-rutter committed:
Corrected errors in border radii values
